packages/p2p/src/components/modal-manager/modals/my-ads-floating-rate-switch-modal/my-ads-floating-rate-switch-modal.tsx

```diff
@@ -31,25 +28,14 @@ const MyAdsFloatingRateSwitchModal = () => {
                             secondary
                             type='button'
                             onClick={() =>
-                                my_ads_store.toggleMyAdsRateSwitchModal(
-                                    my_ads_store.selected_ad_type,
-                                    !floating_rate_store.reached_target_date
-                                )
+                                my_ads_store.toggleMyAdsRateSwitchModal(selected_ad_type, !reached_target_date)
                             }
                             large
                         >
-                            <Localize
-                                i18n_default_text={
-                                    floating_rate_store.reached_target_date ? 'Cancel' : "I'll do this later"
-                                }
-                            />
+                            <Localize i18n_default_text={reached_target_date ? 'Cancel' : "I'll do this later"} />
```

Comment:
wait does this translate properly? cause when we do ternaries to handle the string, other cases it doesn't translate the string since it doesn't recognise it, could you check